contrib/sssd.spec.in

```diff
@@ -1137,8 +1137,8 @@ getent passwd sssd >/dev/null || useradd -r -g sssd -d /run/sssd -s /sbin/nologi
 %__rm -f %{mcpath}/initgroups
 %__rm -f %{mcpath}/sid
 %__chown -f %{sssd_user}:%{sssd_user} %{dbpath}/* || true
-%__chown -f %{sssd_user}:%{sssd_user} %{_sysconfdir}/sssd/sssd.conf || true
-%__chown -f -R %{sssd_user}:%{sssd_user} %{_sysconfdir}/sssd/conf.d || true
+%__chown -f -R :%{sssd_user} %{_sysconfdir}/sssd || true
```

Comment:
To stress we don't enforce user.

Reply:
I don't know actually.
If you think explicit 'root:sssd' is better and others do not object, I'm find with that.

Reply:
I think it would be good to have defined conditions after installation. If distributions want to change it, they can, but it would be good to know the default.

Reply:
Done.